docs/reference/python/katana.local.datastructures.rst

```diff
@@ -3,6 +3,4 @@ Data Structures
 ===============
 
 .. automodule:: katana.local.datastructures
-   :members:
-   :undoc-members:
    :special-members: __init__, __iter__, __getitem__, __setitem__, __len__
```

Comment:
I noticed these special members were deleted elsewhere. Why are they included here?

Reply:
They should have been left in the graph case. See above.

Reply:
The additional special members are left here because some of the data structures have these special methods and they should be documented. I wanted to include all of these in the default, but that causes a huge number of warnings because these methods are then effectively listed on all classes even those that do not have these methods which is (reasonably) a warning in sphinx.